haiku-apps/nanodot/nanodot-1.1b.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A minimal pattern-based MIDI sequencer"
+DESCRIPTION="nanodot is a rather minimal pattern-based MIDI sequencer \
+originally written for BeOS R3. Its main purpose in life was/is to teach me \
+(and hopefully others) the basics of writing MIDI apps for BeOS; it makes for \
+a nice drum-programming 'scratchpad', and is a generally entertaining way to \
+waste a few hours."
+HOMEPAGE="https://github.com/HaikuArchives/NanoDot"
+COPYRIGHT="1999 Eric Moon"
+LICENSE="unknown"
+REVISION="1"
+srcGitRev="80cbeedb3f86ddad8a92bfcf7212704f1528da0f"
+SOURCE_URI="https://github.com/HaikuArchives/NanoDot/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="6cd316390ec1939f7b8e21b1495ec9fffccc3d859b8d0caddb5ca6b6b56399cc"
+SOURCE_FILENAME="NanoDot-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="NanoDot-$srcGitRev"
+PATCHES="nanodot-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	nanodot = $portVersion
+	app:nanodot = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
```

Comment:
Oops, I had missed this one :)
We prefer to not have a blank line between `BUILD_REQUIRES` and `BUILD_PREREQUIRES`.
Source: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Regarding the license, I forgot to tell you that you need to create a subdirectory called "licenses" (in `haikuports/haiku-apps/nanodot`) and create in that `licenses` directory a file called "unknown". For your convenience, after you have remove line 31 if your recipe, you can copy/paste this to your terminal:
```
mkdir licenses
touch licenses/unknown
git add licenses/unknown
git commit --amend licenses/unknown nanodot-1.1b.recipe
git push -f
```
Remarks: you don't need to use `--amend`. This is only a trick to avoid having a separate commit. There is a downside, however: when you "git push" you need to use `--force` (or `-f`), but this doesn't hurt :)
(Sorry if all this was obvious.)

Thanks!

Reply:
Oops, sorry @Oyami-Srk and @pulkomandy. I had missed your recent messages.

@Oyami-Srk, please ignore what I wrote about creating a file called "uknown".